src/e2e/__tests__/iOS/sample.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @jest-environment ./src/e2e/webdriverio-environment.js
+ */
+
+import { Browser } from 'webdriverio'
+import getEditingText from '../../../test-helpers/e2e-helpers/mobile/getEditingText'
+import waitForElement from '../../../test-helpers/e2e-helpers/mobile/waitForElement'
+import newThoughtGesture from '../../../test-helpers/e2e-helpers/mobile/gestures/newThoughtGesture'
+import paste from '../../../test-helpers/e2e-helpers/mobile/paste'
+import waitForEditable from '../../../test-helpers/e2e-helpers/mobile/waitForEditable'
+import clickThought from '../../../test-helpers/e2e-helpers/mobile/clickThought'
+import clickWithOffset from '../../../test-helpers/e2e-helpers/mobile/clickWithOffset'
+// import execute from '../../../test-helpers/e2e-helpers/mobile/execute'
+import tapReturnKey from '../../../test-helpers/e2e-helpers/mobile/tapReturnKey'
```

Comment:
What do you think about housing the e2e test helpers in `/src/e2e/helpers`? They are currently a bit far away.

Reply:
Yes, good idea.